tools/sitemap/main.py

```diff
@@ -87,6 +88,51 @@ def write_place_url(place_type):
     time.sleep(10)
 
 
+def write_priority_places_sitemap():
+  """Write a custom sitemap for SEO testing.
+  
+  Writes a sitemap with 50 US states, Washington D.C., and the top 100
+  US cities by population.
+  """
+  # Get US states and Washington DC
+  sparql = '''
```

Comment:
curious why not fetch State the same way as City, then can merge logic for both of them

Reply:
The ranking call for States will also return places like Puerto Rico, which has multiple dcids and place types. I felt it was easier to match the way the other sitemaps are built instead of filtering the ranking call.

Should I switch to using a call to /api/ranking for State to match City?

Reply:
This works for now. If you add more changes, worth to consolidate.